README.md

```diff
@@ -393,14 +393,13 @@ var name = 'ubuntu-http';
 zone.createVM(name, { os: 'ubuntu' }, function(err, vm, operation) {
   // `operation` lets you check the status of long-running tasks.
 
-  operation.on('complete', function(err, metadata) {
-    if (!err) {
-      // Virtual machine created!
-    }
+  operation.on('complete', function(metadata) {
+    // Virtual machine created!
   });
 });
 ```
 
+You can read more about opeation events [here](https://github.com/GoogleCloudPlatform/gcloud-node/blob/276c847669470c3639eb78d7d87a09db7a9b5c18/lib/compute/operation.js#L83).
```

Comment:
Sorry, I think we should skip this link. It goes against the pattern of our other API doc sections. In general, if a user wants more info about an API, they can click the API or Official Documentation links above the code example.